osu.Game/Rulesets/Edit/HitObjectSelectionBlueprint.cs

```diff
@@ -9,12 +9,12 @@
 
 namespace osu.Game.Rulesets.Edit
 {
-    public abstract class OverlaySelectionBlueprint : SelectionBlueprint<HitObject>
+    public abstract class HitObjectSelectionBlueprint : SelectionBlueprint<HitObject>
     {
         /// <summary>
-        /// The <see cref="DrawableHitObject"/> which this <see cref="OverlaySelectionBlueprint"/> applies to.
+        /// The <see cref="DrawableHitObject"/> which this <see cref="HitObjectSelectionBlueprint"/> applies to.
         /// </summary>
-        public readonly DrawableHitObject DrawableObject;
+        public DrawableHitObject DrawableObject { get; internal set; }
```

Comment:
I think this is fine, but probably needs some xmldoc to explain when this property generally becomes available for use.

Alternatively marking as `NotNull`? But I'm not sure how much sense that makes (it won't migrate well to `nullable` since it *can* actually be null depending on what is managing/constructing the instance).

![20210518 162217 (JetBrains Rider)](https://user-images.githubusercontent.com/191335/118609028-4f493380-b7f5-11eb-8863-33c778517ef1.png)